simple/stats/config.md

```diff
@@ -47,6 +56,12 @@ entities to dcids of that type.
 
 The list of column names to be ignored by the importer, if any.
 
+#### `provenance`
+
+The provenance (name) of this input file. 
```

Comment:
"provenance" is somewhat of a jargon, so perhaps can say it often maps to a dataset from a source?  Maybe give examples of: WorldBank (source) and WorldDevelopmentIndicators (dataset)

Reply:
Done.